src/views/settings.html

```diff
@@ -3,168 +3,155 @@
-                name="enableAdjacentTabs"
-                id="settings_enableAdjacentTabs"
-              />
-              <label class="form-check-label" for="settings_enableAdjacentTabs">
-                The new tab should be next to the current tab
-              </label>
-            </div>
-          </li>
-          <li class="list-group-item">
-            <div class="form-check">
-              <input
-                type="checkbox"
-                class="form-check-input"
-                name="openGroupsInNewWindow"
-                id="settings_openGroupsInNewWindow"
-              />
-              <label
-                class="form-check-label"
-                for="settings_openGroupsInNewWindow"
-              >
-                Open group tabs in new window
-              </label>
-            </div>
-          </li>
-          <li class="list-group-item">
-            <div class="form-check">
-              <input
-                type="checkbox"
-                class="form-check-input"
-                name="enableOptionsMenuItem"
-                id="settings_enableOptionsMenuItem"
-              />
-              <label
-                class="form-check-label"
-                for="settings_enableOptionsMenuItem"
-              >
-                Show link to extension options at the end of context menu
-              </label>
-            </div>
-          </li>
-        </ul>
+          {{#items}}
+            <li class="list-group-item">
+              <div class="form-check">
+                <input
+                  type="checkbox"
+                  class="form-check-input"
+                  name="{{key}}"
+                  id="settings_{{key}}"
+                />
+                <label
+                  class="form-check-label"
+                  for="settings_{{key}}"
+                >
+                  {{label}}
+                </label>
+              </div>
+            </li>
+          {{/items}}
+          </ul>
+        </script>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-6">
+        <h2 class="mb-3 mt-4">Merge Settings Options</h2>
+        <div id="merge-options">Loading...</div>
+        <script id="template_merge-options" type="x-tmpl-mustache">
+          <ul class="list-group">
+            {{#items}}
+              {{!-- The checkbox fields --}}
+              {{#isCheckbox}}
+                <li class="list-group-item">
+                  <div class="form-check">
+                    <input
+                            type="checkbox"
+                            class="form-check-input"
+                            name="{{storeKey}}"
+                            id="settings_{{storeKey}}"
+                    />
+                    <label class="form-check-label" for="settings_{{storeKey}}">
+                      {{label}}
+                    </label>
+                  </div>
+                </li>
+              {{/isCheckbox}}
```

Comment:
Note that you can define a Template at the bottom of the page (or at any part of the page) a then use it to _populate_ several parts of the page. Like:

```
var genericTemplate = document.getElementById("some-template")).innerHTML;

var sectionOne = Mustache.render(genericTemplate , dataForSection1);
document.getElementById("section_one").innerHTML = sectionOne;

var sectionTwo = Mustache.render(genericTemplate , dataForSection2);
document.getElementById("section_two").innerHTML = sectionTwo;
```

> I'm mentioning this this because `template_merge-options` and `template_config` have copy/pasted code between them.
>
> Also note that this is what we currently do for CBC, RCA and NWI: we defined two generic templates (one for configuration values and other for queries) that we later uses on each of these three tabs.